x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -164,7 +164,7 @@ export class RulesClient {
   public bulkDisableRules = (options: BulkDisableRulesRequestBody) =>
     bulkDisableRules(this.context, options);
 
-  public updateApiKey = (options: { id: string }) => updateApiKey(this.context, options);
+  public updateApiKey = (params: { id: string }) => updateApiKey(this.context, params);
```

Comment:
let's rename this to `updateRuleApiKey` so  `update_api_key.ts` -> `update_rule_api_key.ts`